src/MainWindow.ui

```diff
@@ -1664,7 +1664,7 @@ You can drag SQL statements from the Schema column and drop them into the SQL ed
     <string>Load a working session from a file</string>
    </property>
    <property name="whatsThis">
-    <string>This button lets you open a DB4S project file</string>
+    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This button lets you open a DB Browser for SQLite project file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
The `DB4S` -> `DB Browser for SQLite` bit here is easy too.  Not sure about the additional html tags though, are they on purpose?  It's fine if they are, I'm just not sure, so asking. :smile: